lib/srv/usermgmt_test.go

```diff
@@ -506,8 +488,17 @@ func Test_UpdateUserGroups_Drop(t *testing.T) {
 	assert.ElementsMatch(t, append(userinfo.Groups, types.TeleportDropGroup), backend.users["alice"])
 	assert.NotContains(t, backend.users["alice"], types.TeleportKeepGroup)
 
-	// Updates with KEEP mode should convert the ephemeral user
+	// Do not convert the managed user to static.
```

Comment:
Considering this assertion, what happens at the end of this flow?

1. User is logged into a host with a `drop` user
2. An event is received creating a static user of the same name
3. Creation fails because there's already a managed user present (although temporarily)
4. The user disconnects and the drop user is cleaned up
5. What happens here? Is the static user eventually created when the agent restarts?

Reply:
Yes, or when the SHU resource is next updated.